app_dart/lib/src/request_handlers/flaky_handler_utils.dart

```diff
@@ -113,11 +114,13 @@ class IssueUpdateBuilder {
     required this.statistic,
     required this.threshold,
     required this.existingIssue,
+    required this.bucket,
   });
 
   final BuilderStatistic statistic;
   final double threshold;
   final Issue existingIssue;
+  final String bucket;
```

Comment:
Would be cleaner to use enum instead

Reply:
or just a boolean if it is not expected to increase

Reply:
We may support `try` in the future. Updated with enum.